test/EndToEndTests/Tests/Client/Build.Desktop/PluggableFormatServiceTests/PluggableFormatServiceTests.cs

```diff
@@ -287,14 +289,17 @@ public void InvokeAvroAction()
 
         private static void SetAvroMediaTypeResolver(HttpWebRequestMessage requestMessage)
         {
-            requestMessage.Container = ContainerBuilderHelper.BuildContainer(builder =>
-                builder.AddService<ODataMediaTypeResolver, AvroMediaTypeResolver>(ServiceLifetime.Singleton));
+            IServiceCollection services = new ServiceCollection();
+            services.AddDefaultODataServices().BuildServiceProvider();

```

Comment:
Do you need this line given that you're calling `services.AddDefaultODataServices().BuildServiceProvider();` on the line that follows?